patches/minecraft/net/minecraft/client/network/play/NetworkPlayerInfo.java.patch

```diff
@@ -1,10 +0,0 @@
---- a/net/minecraft/client/network/play/NetworkPlayerInfo.java
-+++ b/net/minecraft/client/network/play/NetworkPlayerInfo.java
-@@ -50,6 +50,7 @@
-    }
- 
-    protected void func_178839_a(GameType p_178839_1_) {
-+      net.minecraftforge.client.ForgeHooksClient.onClientChangeGameMode(this, this.field_178866_b, p_178839_1_);
```

Comment:
Not sure why this line is being removed?

Reply:
Again, not sure why that was also touched